pkg/util/redis.go

```diff
@@ -28,12 +25,9 @@ func NewRedis(conf *config.Redis, db int) redis.UniversalClient {
 }
 
 func NewRedisAsynq(conf *config.Redis, db int) asynq.RedisConnOpt {
-	if conf.Address != "" {
-		conf.Addresses = append(conf.Addresses, conf.Address)
-	}
-	if len(conf.Addresses) == 1 {
+	if len(conf.GetAddresses()) == 1 {
```

Comment:
I'd save `conf.GetAddresses()` call result into the variable, as it is called multiple times in that function.

Reply:
done